runtime/jvmti/jvmtiStackFrame.c

```diff
@@ -40,14 +40,15 @@ jvmtiGetStackTrace(jvmtiEnv* env,
 	jvmtiError rc;
 	J9VMThread * currentThread;
 	jint rv_count = 0;
+	J9InternalVMFunctions* vmFuncs = vm->internalVMFunctions;
 
 	Trc_JVMTI_jvmtiGetStackTrace_Entry(env);
 
 	rc = getCurrentVMThread(vm, &currentThread);
 	if (rc == JVMTI_ERROR_NONE) {
 		J9VMThread *targetThread = NULL;
```

Comment:
`vmFuncs` does not get used outside this `if` statement.
```suggestion
		J9VMThread *targetThread = NULL;
		J9InternalVMFunctions* vmFuncs = vm->internalVMFunctions;
```